property/index.js

```diff
@@ -166,3 +156,68 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
+  var isSupported = typeChoices.indexOf(def.type) !== -1;
+  debug('property type "%s" >> supported: %s', def.type, isSupported);
+
+  if (typeof value === 'string' && isSupported) {
+    switch ((def.type || '').toLowerCase()) {
```

Comment:
@bajtos: This is beyond the scope of this fix and should be part of a more comprehensive story that should address the user interaction error handling. Currently, there's no such error handling impl at all to prevent unacceptable input and give end-user another chance to input correct values, based on some explicit console warnings/recommendations